README.md

```diff
@@ -207,3 +208,51 @@ The attribute will render as is on the `<form>` element
 ```html
 <form class="libre-form libre-form-1" data-custom-attr="contactme">
 ```
+
+## Importing forms from a predefined HTML template
+
+Sometimes a project might require static forms which are not supposed to
+be editable in the admin panel.
+
+This plugin allows you to define HTML forms in your project source code
+and import them into the form admin for specific forms.
+
+### Creating a static HTML template
+
+The simplest way is to create a HTML5 file and read its contents. Other
+options include using Twig to render HTML templates.
+
+Remember: WPLF will insert `form` tags on its own, meaning you only have
```

Comment:
Is this necessary? I didn't read this documentation when I tried it out and it removed the form tags for me.

Reply:
Some people might have the idea of adding their own attributes to the `form` tag and they will get mad when they don't appear in the form for no clear reason. I would rather have that documented as the import feature is aimed towards developers who might assume the `form` tags are imported as is.

Reply:
Makes sense. If they want to have custom attributes, they should add those to the shortcode used to print the form.